fitnessllm_dataplatform/stream/strava/services/api_interface.py

```
@@ -0,0 +1,128 @@
+            grant_type: Grant type.
+
+        Returns:
+            dict: Access token.
+        """
+        payload = {
+            "client_id": client_id,
+            "client_secret": client_secret,
+            "code": code,
+            "grant_type": grant_type,
+        }
+        response = requests.post(StravaURLs.AUTH_URL.value, data=payload)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)